nf_core/utils.py

```diff
@@ -15,7 +15,33 @@
 import subprocess
 import sys
 import time
+from distutils import version
+
+try:
+    # Python 3 imports
+    from urllib.request import urlopen
+except ImportError:
+    # Python 2 imports
+    from urllib2 import urlopen
 
+def fetch_latest_version(source_url='https://nf-co.re/version'):
+    """
+    Get the latest version of nf-core
+    """
+    response = urlopen(source_url, timeout=1)
```

Comment:
I didn't think of this until just now, but we should totally just use the `poll_nfcore_web_api()` function that we already have further down in this file..

I'll need to update the website to return JSON, but that should be easy.